cmd/server.go

```diff
@@ -25,9 +26,42 @@ func startServer(c *grumble.Context) error {
 }
 
 func stopServer(c *grumble.Context) error {
-	panic("implement me")
+	if len(c.Args) == 0 {
+		if db == nil {
+			err := errors.New("no server to stop")
+			fmt.Println("flydb stop error: ", err)
+			return err
+		}
+		err := db.Close()
+		if err != nil {
+			fmt.Println("flydb stop error: ", err)
+			return err
+		}
+		fmt.Println("flydb stop success")
+		db = nil
+	}
+	return nil
 }
 
 func cleanServer(c *grumble.Context) error {
```

Comment:
DB Clean should like this method, it should remove `/tmp/*****.data` and `/tmp/.FlyDB_Cli.history` file
![image](https://github.com/ByteStorage/FlyDB/assets/88550136/3ef24322-7b90-44a8-b0ac-3b3fa045ffd4)